test_runner/src/main/kotlin/ftl/domain/RunTestAndroid.kt

```diff
@@ -38,7 +40,7 @@ operator fun RunTestAndroid.invoke() {
     if (dryRun) {
         MockServer.start()
     }
-
+    val prepareStopWatch = StopWatch().run { start() }
```

Comment:
is there a reason why it is not simply:
```suggestion
    val prepareStopWatch = StopWatch().start()
```
The start function returns a instance of this. I do not think the .run is needed here.

Reply:
my mistake, fixed. Thanks!